app/views/HonestyDeclarationPage.scala.html

```diff
@@ -83,6 +83,8 @@ <h1 class="govuk-heading-l">@messages("honestyDeclaration.heading")</h1>
                 }
             } else if(appealType.contains(PenaltyTypeEnum.Late_Payment.toString) || appealType.contains(PenaltyTypeEnum.Additional.toString)) {
                     @getMessage("honestyDeclaration.lpp.li.2")
+            } else if(userRequest.isAgent && userRequest.session.get(SessionKeys.whoPlannedToSubmitVATReturn).contains("agent") && userRequest.session.get(SessionKeys.whatCausedYouToMissTheDeadline).contains("client")) {
```

Comment:
We will need to create a ticket to make this view contains less logic, I'll approve it as it's not fair for you to do all that rework here